main/res/values/strings.xml

```diff
@@ -691,15 +691,15 @@
     <string name="settings_restart">c:geo needs to be restarted now to bring loaded settings into effect</string>
 
     <string name="settings_info_offline_maps_title">Info on Offline Maps</string>
-    <string name="settings_info_offline_maps">c:geo supports maps for offline use. You can download maps from Mapsforge or even create your own maps from OSM data. You need to select the offline map directory first to get offline map support.</string>
+    <string name="settings_info_offline_maps">c:geo supports maps for offline use. You can download maps from Mapsforge or even create your own maps from OSM data. You need to select the offline map folder first to get offline map support.</string>
```

Comment:
Download from Mapsforge?
How about: `c:geo supports maps for offline use. You can obtain maps from different providers or even create your own maps (from OSM data). First you have to select the offline map folder in which the offline maps are to be stored.`

Reply:
You are right, this strings is quite old.